src/server/middleware/util.js

```diff
@@ -83,7 +86,7 @@ async function errorMiddleware(ctx, next) {
 }
 
 function handleCouchError(ctx, e, secure) {
-  if (e.scope !== 'couch') {
+  if (e.name !== 'HTTPError') {
```

Comment:
does this result in the exact same behavior as with the nano library?

Reply:
I'm not sure. Don't really know how to verify